File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q36.js
import { question } from "readline-sync";

function main(){
    //Input

    const years = parseInt(question("Enter the age in years: "))
    const months = parseInt(question("Enter the age in months: "))
    const days = parseInt(question("Enter the age in days: "))

    //Processing

    const total_days = (years * 365 + months * 30 + days)

    //Output
    console.log(`The age entered converted to days is equal to: ${total_days}`)

}
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q36.py
def main():
    #Input

    years = int(input("Enter the age in years: "))
    months = int(input("Enter the age in months: "))
    days = int(input("Enter the age in days: "))

    #Processing

    total_days = ((365 * years) + (months * 30) + days)

    #Output
    print(f"The total days of the entered age in days is equal to: {total_days}")

main()